README.md

```diff
@@ -7,6 +7,9 @@ Material Components for Tizen .NET help developers execute [Material Design](htt
 - **MActivityIndicator**
   A circular progress indicator that supports determinate and indeterminate circular progress indicators. ([guideline](https://material.io/design/components/progress-indicators.html#circular-progress-indicators))
 
+- **MConformant**
+  A conformant allow the user to use a floating action button.
```

Comment:
You may be intended to mention like below?
"A material conformant allows ~~ " 
,or just "A conformant allows ~~"

Reply:
@JoonghyunCho Fixed it. Thank you.